pkg/schema/schema.go

```diff
@@ -241,6 +249,9 @@ func validateKind(kind string, fileBytes []byte) error {
 		if err != nil {
 			return err
 		}
+		if validateExamplesAnnotations(&csv) != nil {
```

Comment:
I don't think this is actually running - I tried to mess up an example and it still passed validation when loading ☹️